site/content/en/guides/producer/init/_index.md

```diff
@@ -74,3 +78,9 @@ Recommended: tag the commit as a release
 git tag nginx/v0.1.0
 git push nginx/v0.1.0
 ```
+
```

Comment:
Nit: Update table of contents with new section

Reply:
Fixed